packages/hurl/src/runner/hurl_file.rs

```diff
@@ -220,14 +220,25 @@ pub fn run(
                 // an error. If we want to treat it as an error, we've to add it to the current
                 // `entry_result` errors, and optionally deals with retry if we can't write to the
                 // specified path.
-                if !runner_options.context_dir.is_access_allowed(&output) {
-                    let inner = RunnerError::UnauthorizedFileAccess {
-                        path: PathBuf::from(output.clone()),
-                    };
-                    let error = runner::Error::new(entry.request.source_info, inner, false);
-                    logger.warning(&error.fixme());
-                } else if let Err(error) = entry_result.write_response(output) {
-                    logger.warning(&error.fixme());
+
+                let authorized = if let Output::File(filename) = output.clone() {
```

Comment:
the `output.clone() ` doesn' seems necessay no ?

Reply:
I was expected this comment :-)
I'll try to remove it